utils/credentialsUtils.js

```diff
@@ -0,0 +1,137 @@
+import { v4 as uuidv4 } from 'uuid';
+import { toast } from 'sonner';
+import { createBankIdCredential } from '../_credentials/quotient';
+import { createCreditScoreCredential } from '../_credentials/equinet';
+import { createBiometricsCredential } from '../_credentials/forsur';
+import { postRequest } from './request';
+
+const dockUrl = 'https://api-testnet.dock.io';
+
+async function encryptAndPrepareMessage(senderDID, receiverDID, credentialsSubject) {
+  console.log('encryptAndPrepareMessage');
+
+  const encryptionPayload = {
+    senderDid: senderDID,
+    recipientDids: [receiverDID],
+    type: 'issue',
+    payload: {
+      domain: 'api.dock.io',
+      credentials: credentialsSubject
+    }
+  };
+
+  console.log('encryptionPayload:', encryptionPayload);
+
+  const data = await postRequest(`${dockUrl}/messaging/encrypt`, encryptionPayload);
+
+  console.log('didComm Message:', data);
+
+  return data.data.jwe;
+}
+
+const signedCredential = async (receiverDid) => {
+  const enrollmentId = `${uuidv4()}`;
+
+  const bankIdData = {
+    receiverDid,
+    receiverName: 'John Smith', // Name input
+    receiverAddress: '123 Main St, Anytown, CA 90210',
+    enrollmentId
+  };
+
+  const biometricsData = {
+    receiverDid,
+    enrollmentId,
+    biometricData: 'some_biometrics_data',
+    enrollmentDate: '2024-02-05',
+    validTo: '2024-03-05',
+  };
+
+  const creditScoreData = {
+    receiverDid,
+  };
+
+  const bankIdCredential = createBankIdCredential(bankIdData);
+  const creditScoreCredential = createCreditScoreCredential(creditScoreData);
+  const biometricsCredential = createBiometricsCredential(biometricsData);
+
+  const credentials = [
+    bankIdCredential,
+    creditScoreCredential,
+    biometricsCredential
+  ];
+
+  const issuedCredentials = await Promise.all(credentials.map(async (credential) => {
+    const credentialData = await postRequest(credential.url, credential.body);
+    return credentialData.data;
+  }));
+
+  return issuedCredentials;
+};
+
+export const issueCredentials = async (
+  receiverDID,
+  setIsLoading,
+  setIsSuccess
+) => {
+  const responses = [];
+
+  try {
+    const issuedCredentials = await signedCredential(receiverDID);
+
+    const senderDIDMapping = [
+      'did:dock:5HKkVpaciu1RArV13E7ig3i84JtiMTcwoXoHPZ8VMrBUYJ4w',
```

Comment:
Can we use some constants for these 3 values, I saw these values are duplicated in `/_credentials/` for issuers. I suggest for easy configurations to use `.env` variables for them so in case we need to change API keys to be able to do this w/o the need of a separate PR to modify these values.

Reply:
@dhipgraby You updated these values with ones from .env, which is good, but the values from `/_credentials/*` are still hard-coded. I suggest searching for all occurrences of `did:dock` values and using the right .env values.

Reply:
All credentials also changed to use process.env.
Thanks